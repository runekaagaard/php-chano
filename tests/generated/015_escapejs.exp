\u0022double quotes\u0022 and \u0027single quotes\u0027