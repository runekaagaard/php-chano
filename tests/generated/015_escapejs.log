
---- EXPECTED OUTPUT
\u0022double quotes\u0022 and \u0027single quotes\u0027
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::escapejs() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/015_escapejs.php on line 4
---- FAILED
