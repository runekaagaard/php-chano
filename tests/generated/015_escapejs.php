<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => '"double quotes" and \'single quotes\'')));
foreach ($items as $i) echo $i->input->escapejs();
