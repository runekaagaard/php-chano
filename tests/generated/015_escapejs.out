Fatal error: Call to undefined method DtlIter::escapejs() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/015_escapejs.php on line 4